src/boilerplate.rs

```diff
@@ -1,22 +1,87 @@
 use std::fs;
+use std::path::Path;
 
 use crate::html;
 use crate::spec::Spec;
 
+// Retrieve boilerplate file with doc (metadata).
+fn retrieve_boilerplate(doc: &mut Spec, name: &str) -> String {
+    retrieve_boilerplate_with_info(name, doc.md.group.clone(), doc.md.raw_status.clone())
+}
+
+// Retrieve boilerplate file with group and status.
+fn retrieve_boilerplate_with_info(
+    name: &str,
+    group: Option<String>,
+    status: Option<String>,
```

Comment:
Do these really need to be `Option<String>`? It seems there's no reason for that and they should just be `Option<&str>`.